src/cmd/api/router/router.go

```diff
@@ -72,6 +72,7 @@ func NewRouter(config config.Config, clients *Clients, uuidGen *UUIDGenerator, D
 
 	v1.PathPrefix("/videos/{id}/streams/master.m3u8").Handler(controllers.VideoGetMasterHandler{S3Client: clients.S3Client, UUIDGen: uuidGen.UUIDGen}).Methods("GET")
 	v1.PathPrefix("/videos/{id}/streams/{quality}/{filename}").Handler(controllers.VideoGetSubPartHandler{S3Client: clients.S3Client, UUIDGen: uuidGen.UUIDGen, ServiceDiscovery: clients.ServiceDiscovery}).Methods("GET")
+	v1.PathPrefix("/videos/transformer").Handler(controllers.VideoTransformerListHandler{ServiceDiscovery: clients.ServiceDiscovery}).Methods("GET")
```

Comment:
To match with other endpoints, what do you think about rename this endpoint by something like transformers/list?